.github/workflows/ci.yaml

```diff
@@ -36,16 +35,13 @@ jobs:
               id: composer-cache
               run: echo "::set-output name=dir::$(composer config cache-files-dir)"
 
-            - uses: actions/cache@v1
-              id: cache-composer
+            - name: Composer - Set cache
+              uses: actions/cache@v2
               with:
                   path: ${{ steps.composer-cache.outputs.dir }}
-                  key: composer-php.${{ matrix.php }}-${{ github.sha }}
-                  restore-keys: composer-php.${{ matrix.php }}-
-
-            - name: Composer - Create cache directory
-              run: mkdir -p /home/runner/.composer/cache
-              if: steps.cache-composer.outputs.cache-hit != 'true'
+                  key: php-${{ matrix.php }}-symfony-${{ matrix.symfony }}-composer-${{ hashFiles('**/composer.json **/composer.lock') }}
```

Comment:
We don't have `composer.lock` in plugin repo. Is this really needed ?

Reply:
It doesn't change anything but it's more readable so I modify.